mlflow/pipelines/steps/ingest/__init__.py

```diff
@@ -76,8 +76,8 @@ def _run(self, output_directory: str) -> BaseCard:
         ingested_dataset_profile = None
         if not self.skip_data_profiling:
             _logger.info("Profiling ingested dataset")
-            ingested_dataset_profile = get_pandas_data_profile(
-                ingested_df, "Profile of Ingested Dataset"
+            ingested_dataset_profile = get_pandas_data_profiling_comparison_view(
+                [["Profile of Ingested Dataset", ingested_df]]
```

Comment:
There's only 1 dataset in `ingest`, right? Why do we need to use the comparison view? consistency?

Reply:
We have logic in pandas_renderer that splits the view between compare view v/s single view. So to keep the API clean and have the split logic in one place, we can just pass in an array and let pandas_renderer handle that.